package.json

```diff
@@ -124,6 +130,9 @@
         "stylelint-scss": "3.3.1",
         "stylelint-webpack-plugin": "0.10.5",
         "terser-webpack-plugin": "1.2.3",
+        "ts-jest": "24.1.0",
+        "ts-loader": "6.2.1",
+        "typescript": "3.6.4",
```

Comment:
3.7 has just been released. let's use that.

Reply:
Done! I Added extension "*.tsx" to .eslintrc.json for storybook.